node/src/main/scala/com/wavesplatform/Application.scala

```diff
@@ -42,14 +43,13 @@ import kamon.Kamon
 import kamon.influxdb.InfluxDBReporter
 import kamon.system.SystemMetrics
 import monix.eval.{Coeval, Task}
-import monix.execution.Scheduler
-import monix.execution.Scheduler._
 import monix.execution.schedulers.SchedulerService
+import monix.execution.{Scheduler, UncaughtExceptionReporter}
 import monix.reactive.Observable
 import monix.reactive.subjects.ConcurrentSubject
 import org.influxdb.dto.Point
 import org.slf4j.LoggerFactory
-
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Shouldn't `global` be wrapped too?

Reply:
Seems like there's no need to: we never explicitly shut down global execution context.